net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild

```diff
@@ -3,32 +3,24 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{6,7} )
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
3.9? If deps in the way, file bugs if they don't exist -- or even fix them too ;)

Reply:
Deps are okay, py3.9 is added

Reply:
If tests pass and it works, cool.

Reply:
Works with 3.9, there are no tests AFAICS